utils/make-phar.php

```diff
@@ -216,10 +216,20 @@ function get_composer_versions( $current_version ) {
 	add_file( $phar, $file );
 }
 
-// other files
 $finder = new Finder();
+
 $finder
 	->files()
+	->in(EE_VENDOR_DIR . '/easyengine/site-command/templates')
```

Comment:
How about you make it generic so that any command can add template files?

Reply:
If we would want to add mustache templates in another command, we just have to copy paste the above line and change path. I've narrowed down the path intentionally to make sure we're not including any mustache file accidentially(where we're not supposed to). 

Also in case there's any confusion, we don't have to worry about 3rd party commands as they won't be included in phar